### PR TITLE
[Kotlin Migration] CustomTabActivityHelperTest

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt [hunk 1]
```diff
@@ -13,87 +13,83 @@
...
+
+@KotlinCleanup("`when` -> whenever")
+@KotlinCleanup("remove @NonNull")
+@RunWith(RobolectricTestRunner::class)
+class CustomTabActivityHelperTest {
     @Before
-    public void before() {
-        CustomTabActivityHelper.resetFailed();
+    fun before() {
+        CustomTabActivityHelper.Companion.resetFailed()
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt:45]: Is the .Companion needed?

File: AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt [hunk 2]
```diff
@@ -13,87 +13,83 @@
...
-
-        CustomTabsClient invalidClient = mock(CustomTabsClient.class);
-        doThrow(exceptionToThrow).when(invalidClient).warmup(anyLong());
-        doThrow(exceptionToThrow).when(invalidClient).extraCommand(anyString(), any());
-        doThrow(exceptionToThrow).when(invalidClient).newSession(any());
-        return invalidClient;
+    private fun getClientThrowingSecurityException(): CustomTabsClient {
+        val exceptionToThrow = SecurityException("Binder invocation to an incorrect interface")
+
+        val invalidClient = mock(CustomTabsClient::class.java)
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt:89]: can likely use `mock { }`

File: AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt [hunk 3]
```diff
@@ -13,87 +13,83 @@
...
+        val activity = mock(Activity::class.java)
+        val packageManager = mock(PackageManager::class.java)
 
-        when(activity.getPackageManager()).thenReturn(packageManager);
-        when(packageManager.queryIntentActivities(any(), anyInt())).thenReturn(Collections.emptyList());
+        `when`(activity.packageManager).thenReturn(packageManager)
+        `when`(packageManager.queryIntentActivities(any(), anyInt())).thenReturn(emptyList())
 
-        CustomTabActivityHelper.openCustomTab(activity, null, null, fallback);
+        CustomTabActivityHelper.Companion.openCustomTab(activity, null, null, fallback)
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt:72]: ditto: companion

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks good, one question

File: AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt (hunk 2 above)
Comment (BrayanDSO) [reply AnkiDroid/src/test/java/com/ichi2/compat/customtabs/CustomTabActivityHelperTest.kt:90]: Nice catch

**Review by mikehardy — APPROVED** [review]
:shipit:

**Pull request merged**